templates/default/ckan_properties.ini.erb

```diff
@@ -241,6 +241,13 @@ ckanext.cloudstorage.container_name = <%= node['datashades']['attachments_bucket
 ckanext.cloudstorage.driver_options = use_role
 ckanext.cloudstorage.use_secure_urls = 1
 
+
+ckanext-archiver.archive_dir = /resource_cache
```

Comment:
Where is this path? On disk? If we're writing large amounts to disk, they should be on EFS or at least on the 32GB disk.

Reply:
plan is that we change the archiver to use the uploader and this is the prefix when its pushed to the upload module

we could sym link it to efs for now but yeah its not a good spot due to cost overall.

Reply:
The trouble is that it's not usable with the current value; it would quickly fill the root disk and bring down the application.

Reply:
changed to /data/resource_cache to unblock this, still need to work on ckan archiver to use the uploader system and try pattern /dataset/{id}/resource/{resource_id}/archive/{filename}